src/zope/server/http/tests/test_wsgiserver.py

```diff
@@ -223,7 +241,7 @@ def testDeeperPath(self):
         self.assertEqual(status, 200)
         expect_response = 'URL invoked: http://%s:%d/folder/item' % (
             LOCALHOST, self.port)
-        self.assertEqual(response_body, expect_response)
+        self.assertEqual(response_body, expect_response.encode())
```

Comment:
Known test data; ASCII works fine as the charset.

Unless you would prefer explicit `.encode('ascii')` in all the tests, for stylistic reasons?

Reply:
It's probably not really a big deal since these are tests. 

It's just that I have been conditioned to provide an explicit charset for encoding/decoding; not only is "explicit better than implicit", it saves me the trouble of remembering what the default encoding happens to be (ignoring the fact that theoretically it can be changed), and it's one less exception to the rules ("always specify your charset, *except* in test code when you know the default will decode correctly").

Alternatively, a 'b' prefix on the response literals is just as good. (Except, I know, formatting.)

Reply:
Oh yeah, I have very different internal standards for production code versus test code, but I can see where that can go wrong.